CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -106,6 +106,8 @@ function PlayerHospital:dailyAdviceChecks()
   -- Reset advise flags at the end of the month.
   if day == 28 then
     self.adviser_data.temperature_advice = false
+    self.adviser_data.gps_office = false
+    self.adviser_data.gps_office_1 = false
```

Comment:
Bad names. `gps_office` is way too generic (the message doesn't just say "GP office", the message says to build one and hire a doctor.

The second variable name is also too generic. In addition, using a numeric suffix should generally be avoided.